src/System.Private.CoreLib/shared/System/Globalization/TextInfo.cs

```diff
@@ -439,15 +277,15 @@ internal void ChangeCaseToUpper(ReadOnlySpan<char> source, Span<char> destinatio
                             // this method, so we won't bother vectorizing until data shows us that it's worthwhile to do so.
 
                             uint tempValue = Unsafe.ReadUnaligned<uint>(pSource + currIdx);
-                            if (!Utf16Utility.AllCharsInUInt32AreAscii(tempValue))
+                            if (!Utf16Utility.AllCharsInUIntAreAscii(tempValue))
```

Comment:
Nit: Personally, I'd prefer to see the "32" remain.  We generally refer to the type names in method names like this.